extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/test/java/io/quarkus/resteasy/reactive/server/test/multipart/MultipartOutputResource.java

```diff
@@ -19,6 +21,7 @@
     public static final List<String> RESPONSE_VALUES = List.of("one", "two");
     public static final boolean RESPONSE_ACTIVE = true;
 
+    private static final int ONE_GIGA = 1024 * 1024 * 1024;
```

Comment:
the exception that I (just) pasted in the issue said specifically that at ca 2*10^9, we should test sth like 3G

Reply:
I could reproduce it with 1GB only in my machine. I tried with 3gb and worked fine too. However, I think using 1gb will be better in terms of CI resources (if we keep this test)